### PR TITLE
Moved Mod Manager to the top menu. Reduced sound noise.

### PR DESCRIPTION
**Pull request by KionX**

### CODE REVIEW

File: lua/ui/menus/main.lua [hunk 1]
```diff
@@ -124,9 +124,10 @@ function CreateUI()
             color = menuFontColorAlt,
         },
         {
-            name = '<LOC tooltipui0355>Extras',
-            tooltip = 'mainmenu_extras',
-            action = function() ButtonExtras() end,
+            name = '<LOC _Mod_Manager>',
```
Comment (speed2CZ) [inline lua/ui/menus/main.lua]: There is no reason for this change, most of the time player will get to the mod manager through the lobby anyway. + you kill the option to check credits and eula. So not this way

Comment (KionX) [reply lua/ui/menus/main.lua]: Who is interested in check credits and eula?
They are available in vanilla FA.

**Comment by shalkya** [comment]
removing the sound is good.
To find a middle ground would it be possible to keep "extra" and to just add "mod manager" under ? With some repositioning it should fit.

**Comment by Uveso** [comment]
Making the menu bigger does not fit into the mainscreen:
Mainmenu

**Comment by Uveso** [comment]
Working now!
Menu fits to the screen and is working.

**Pull request merged**